devops/infrastructure/roles/rpc/templates/Caddyfile.j2

```diff
@@ -0,0 +1,3 @@
+{{ public_ip }}.nip.io
+
+reverse_proxy localhost:9944
```

Comment:
Its a a good idea to also proxy the http-rpc port 9933. So maybe have two pathes one for websocket and other for the non websocket endpoint:

{{ public_ip }}.nip.io/ws-rpc  --> reverse_proxy localhost:9944
and
{{ public_ip }}.nip.io/http-rpc -> reverse_proxy localhost:9933

and have these paths be also based in as template variables from `roles/rpc/tasks/main.yml`

Reply:
Done